### PR TITLE
mvn tidy:pom && reformat indentation

### PR DESCRIPTION
**Pull request by stephenc**
@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,85 +1,95 @@
...
+      <name>MIT</name>
+      <url>http://opensource.org/licenses/MIT</url>
+    </license>
+  </licenses>
+
+  <scm>
+    <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+    <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
+    <tag>HEAD</tag>
```
Comment (tfennelly) [inline pom.xml]: From what I can see here, just looks like indentation + move of <licenses> section?

**Comment by tfennelly** [comment]
:bee:

**Comment by varmenise** [comment]
:bee:

**Comment by stephenc** [comment]
@reviewbybees done

**Comment by stephenc** [comment]
(as plugin maintainer) merging as this is a trivial merge

**Pull request merged**